src/main/java/com/gtnewhorizons/navigator/api/util/DrawUtils.java

```diff
@@ -163,30 +239,32 @@ public static void drawLabel(String text, double textX, double textY, int fontCo
             textY /= fontScale;
             GL11.glScaled(fontScale, fontScale, 0);
         }
+
         double dTextX = textX - (double) (int) textX;
         double dTextY = textY - (double) (int) textY;
         double textWidth = fontRenderer.getStringWidth(text);
         double xOffsetL = centered ? -textWidth / 2.0 - 2 : -2;
-        double xOffsetR = centered ? textWidth / 2.0 + 2 : textWidth + 2;
         GL11.glTranslated(dTextX, dTextY, 0.0);
-        drawGradientRect(
-            (int) textX + xOffsetL,
-            (int) textY - 2,
-            (int) textX + xOffsetR,
-            (int) textY + fontRenderer.FONT_HEIGHT + 2,
-            0,
-            bgColor,
-            bgColor);
-        fontRenderer.drawStringWithShadow(
-            text,
-            (centered ? (int) (textX - textWidth / 2.0) : (int) textX),
-            (int) textY,
-            fontColor);
+        drawRect(textX + xOffsetL, textY - 2, textWidth + 2, fontRenderer.FONT_HEIGHT + 2, bgColor, 180);
+        if (fontShadow) {
+            fontRenderer.drawStringWithShadow(
+                text,
+                (centered ? (int) (textX - textWidth / 2.0) : (int) textX),
+                (int) textY,
+                fontColor);
+        } else {
+            fontRenderer
+                .drawString(text, (centered ? (int) (textX - textWidth / 2.0) : (int) textX), (int) textY, fontColor);
+        }
         GL11.glPopMatrix();
     }
 
     public static float[] floats(int rgb) {
         return new float[] { (float) (rgb >> 16 & 255) / 255.0F, (float) (rgb >> 8 & 255) / 255.0F,
             (float) (rgb & 255) / 255.0F };
     }
+
+    public static int[] ints(int rgb, int alpha) {
+        return new int[] { rgb >> 16 & 255, rgb >> 8 & 255, rgb & 255, alpha & 255 };
```

Comment:
nitpick: Visually mark precedence of operators.
```suggestion
        return new int[] { (rgb >> 16) & 255, (rgb >> 8) & 255, rgb & 255, alpha & 255 };
```